test/xspec.bats

```diff
@@ -190,3 +190,10 @@ teardown() {
     [ "$status" -eq 0 ]
     [[ "${lines[3]}" =~ "Testing with" ]]
 }
+
+@test "executing the Saxon XProc harness generates a report with UTF-8 encoding" {
+	run bash -c "java -Xmx1024m -cp ${XMLCALABASH_CP} com.xmlcalabash.drivers.Main -isource=xspec-72.xspec xspec-home=file:///home/travis/build/xspec/xspec/ -oresult=/home/travis/build/xspec/xspec/test/xspec/xspec-72-report.html /home/travis/build/xspec/xspec/src/harnesses/saxon/saxon-xslt-harness.xproc && grep 'charset=UTF-8' /home/travis/build/xspec/xspec/test/xspec/xspec-72-report.html"
```

Comment:
I see your point now. I implemented a bats test that uses Saxon to check the encoding as follows:
```
java -cp ${SAXON_CP} net.sf.saxon.Query -s:${TRAVIS_BUILD_DIR}/test/xspec/xspec-72-result.html -qs:"declare default element namespace 'http://www.w3.org/1999/xhtml'; string(html/head/meta[@http-equiv eq 'Content-Type']/@content) = 'text/html; charset=UTF-8'"
```

An alternative approach to make a more precise bats test with ```grep``` is to use  ```sed``` to select a specific line of the HTML report (line 3 in this case) and pipe it through ```grep``` in order to match the full ```<meta>``` element (not just ```charset=UTF-8```). I report it here for documentation but I have not included it in the latest changes:
```bash
run bash -c "java -Xmx1024m -cp ${XMLCALABASH_CP} com.xmlcalabash.drivers.Main -isource=xspec-72.xspec xspec-home=file://${TRAVIS_BUILD_DIR}/ -oresult=${TRAVIS_BUILD_DIR}/test/xspec/xspec-72-result.html ${TRAVIS_BUILD_DIR}/src/harnesses/saxon/saxon-xslt-harness.xproc && sed '3!d' ${TRAVIS_BUILD_DIR}/test/xspec/xspec-72-result.html | grep '<meta http-equiv=\"Content-Type\" content=\"text/html; charset=UTF-8\" />' "
```